frontend/packages/dev-console/integration-tests/testData/create_user.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+HTPASSWD_FILE="./htpass"
+USERNAME="test"
+USERPASS="test"
+HTPASSWD_SECRET="htpasswd-test-secret"
+
+OC_USERS_LIST="$(oc get users)"
+if echo "${OC_USERS_LIST}" | grep -q "${USERNAME}"; then
+    echo -e "\n\033[0;32m \xE2\x9C\x94 User test already exists \033[0m\n"
+    exit;
+fi
+htpasswd -cb $HTPASSWD_FILE $USERNAME $USERPASS
+
+oc get secret $HTPASSWD_SECRET -n openshift-config &> /dev/null
+
+oc create secret generic ${HTPASSWD_SECRET} --from-file=htpasswd=${HTPASSWD_FILE} -n openshift-config
+
+oc apply -f - <<EOF
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  identityProviders:
+  - name: test
+    challenge: true
+    login: true
+    mappingMethod: claim
+    type: HTPasswd
+    htpasswd:
+      fileData:
+        name: ${HTPASSWD_SECRET}
+EOF
+
+sleep 10s
+oc create clusterrolebinding ${USERNAME}_role1 --clusterrole=developer --user=${USERNAME}
+oc create clusterrolebinding ${USERNAME}_role2 --clusterrole=self-provisioner --user=${USERNAME}
+oc create clusterrolebinding ${USERNAME}_role2 --clusterrole=basic-user --user=${USERNAME}
+
+  output=''
+  count=0
+  sleepDuration=15
+  maxRetries=40  #10 minutes
+  action="started"
+  if [ "$1" == "False" ]; then
```

Comment:
Multiple times this code mentions `$1`. I think this is a left over because it was a function before, or?

Reply:
Removed it